### DISCUSSION TITLE
OLED screen for G14 2024

### DISCUSSION THREAD

**Discussion by luniousty** [seerge/g-helper#2777] [Q&A]
As this new laptop has a OLED screen and probably have burn in issue, does ghelper has the solution to it? Because Armoury Crate has a feature for it. It is like refresh the screen every 3 minutes or something.

**Comment by seerge** [comment]
@luniousty hello, I was answering this multiple times already

AC has 2 settings for that, but they can be also handled by windows directly.

- Hide taskbar checkbox - is something you can toggle in windows settings (right click on taskbar)
- "Pixel shift" checkbox - to my best knowledge it only moves **your wallpaper** few pixels here and there during time. You can just set windows to rotate wallpapers periodically, that would have better effect.

There is a mini-application in AC folder (Asus Oled Shifter) that does move wallpaper - it literally keeps multiple edited copies of your wallpaper there.
